include/odp/api/spec/classification.h

```diff
@@ -103,6 +103,18 @@ typedef union odp_cls_pmr_terms_t {
 		/** Custom layer 3 match rule. PMR offset is counted from
 		 *  the start of layer 3 in the packet. */
 		uint64_t        custom_l3:1;
+		/** IGMP Group address */
+		uint64_t	igmp_grp_addr:1;
+		/** ICMP identifier */
+		uint64_t	icmp_id:1;
+		/** ICMP type */
+		uint64_t	icmp_type:1;
+		/** Source SCTP port */
+		uint64_t	sctp_sport:1;
+		/** Destination SCTP port */
+		uint64_t	sctp_dport:1;
+		/** GTPU Tunnel endpoint identifier */
+		uint64_t	gtpu_teid:1;
```

Comment:
What UDP port number is matched as GTPU?

Reply:
We will send the next version with the definition of GTPU in the comment.